### PR TITLE
Change name of non-expresive variables to expresive one

### LINKED ISSUE #8521: Rename non-expressive variables
**Description of Problem**:
on the files:
 - `.../rasa/core/tracker_store.py `;
 - `.../rasa/utils/io.py`;
 - `/rasa/core/channels/mattermost.py`. 
 
  has some variables with a non expressive name.

**Overview of the Solution**:
Changing variables that are just letters to names that express exactly what the variable represents.

### PR DESCRIPTION
**Pull request by ericoBandeira**
**Proposed changes**:
- Rename non-expressive variables to expressive
- Resolve #8521

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @yennycheung will take a look at it as soon as possible ✨

File: rasa/core/channels/mattermost.py [hunk 1]
```diff
@@ -28,11 +28,11 @@ def token_from_login(cls, url: Text, user: Text, password: Text) -> Optional[Tex
         """Retrieve access token for mattermost user."""
 
         data = {"login_id": user, "password": password}
-        r = requests.post(url + "/users/login", data=json.dumps(data))
-        if r.status_code == 200:
-            return r.headers["Token"]
+        request = requests.post(url + "/users/login", data=json.dumps(data))
```
Comment (koernerfelicia) [inline rasa/core/channels/mattermost.py:31]: This and the `r` below  in `__init__` actually refer to "response", not "request"

Comment (ericoBandeira) [reply rasa/core/channels/mattermost.py]: Thanks @koernerfelicia , I got it wrong, I already made the changes with the correct names!

**Review by ericoBandeira — COMMENTED** [review]
Change named variables

**Comment by yennycheung** [comment]
@koernerfelicia, I see you're already looking at this PR, I'll remove myself as a reviewer.

**Review by koernerfelicia — APPROVED** [review]
LGTM! Thank you @ericoBandeira

**Review by koernerfelicia — CHANGES_REQUESTED** [review]
sorry, -- just realised the code quality test is still failing. Could you take care of this, first? Then it will be good to merge

**Comment by koernerfelicia** [comment]
@ericoBandeira could you take care of the failing tests? Then this will be ready to merge.

**Comment by koernerfelicia** [comment]
Closed because stale.

**Pull request closed without merging**